opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -29,6 +29,7 @@
 #include <opm/parser/eclipse/EclipseState/Schedule/WellProductionProperties.hpp>
 #include <opm/parser/eclipse/EclipseState/Schedule/WellInjectionProperties.hpp>
 #include <opm/parser/eclipse/EclipseState/Schedule/WellPolymerProperties.hpp>
+#include <opm/parser/eclipse/EclipseState/Schedule/WellRFTProperties.hpp>
```

Comment:
The class `WellRFTProperties` is not necessary, this collection only lives in the `iterateScheduleSection`method and the functionality can be handled with `std::pair`:
```C++
std::vector<std::pair<DeckKeywordConstPtr , size_t> > rftProperties;
...
rftPreperties.push_back( std::make_pair( keyword , currentStep ));
....
for (auto rftPair = rftProperties.begin(); rftPair != rftProperties.end(); ++rftPair) {
     DeckKeywordConstPtr keyword = rftPair->first;
     size_t timeStep = rftPair->second;
}
```